Source/JavaWrappers.cpp

```diff
@@ -4,6 +4,7 @@
 #include <android/surface_texture_jni.h>
 #include <android/asset_manager_jni.h>
 #include <android/native_window_jni.h>
+#include <algorithm>
```

Comment:
should also include `vector`

Reply:
It's included in the .h file already, so probably don't need to add it here too?

Reply:
I usually add all headers that are used in the file directly. Headers can change and if vector is removed from another dependency, it will cause this file to break. But, it's not a big deal.

Reply:
Got it, added.